### PR TITLE
Standalone binary

### PR DESCRIPTION
**Pull request by weierophinney**
In seeing some people attempt to use the library, I realized it may be better to ship it as a standalone binary:

- The class it generates has no dependencies.
- Generally, you will generate the preload file once, and then manipulate it to provide additional rules. Once generated, you no longer need the tooling

This patch provides a `bin/phly-opcache-preload` file exposed as a vendor binary in order to allow global installation. When used in this way, the commands become `generate:preload-file` and `generate:ini`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     },
     "require": {
         "php": "~7.4.4",
-        "laminas/laminas-cli": "dev-master as 0.1.0"
+        "laminas/laminas-cli": "dev-master as 0.1.0",
```
Comment (michalbundyra) [inline composer.json]: I think we should drop laminas-cli dependency now.

File: composer.json [hunk 2]
```diff
@@ -14,7 +14,8 @@
...
+        "symfony/console": "^5.0"
```
Comment (michalbundyra) [inline composer.json]: I think we should allow also previous version of symfony/console (maybe the same as we have in laminas-cli symfony/console 4.1 || 5.0)

**Review by michalbundyra — CHANGES_REQUESTED** [review]
@weierophinney please see my comments below, thanks!

**Comment by weierophinney** [comment]
@michalbundyra Noted your points, and pushed a change that adopts your feedback, as well as:

- Removes the `ConfigProvider` and `Module` classes, as they are then no longer relevant.
- Removes verbiage in the `README.md` about usage with laminas-cli.

**Comment by michalbundyra** [comment]
@weierophinney I kinda like that it was available in laminas-cli when laminas-cli is installed, but the problem is that then the module/config-provider must be registered, and it is not something we want to have in the applications. So the changes, you just have made, make sense.

**Review by michalbundyra — APPROVED** [review]
🚢

**Comment by weierophinney** [comment]
> I kinda like that it was available in laminas-cli when laminas-cli is installed, but the problem is that then the module/config-provider must be registered, and it is not something we want to have in the applications.

Exactly. It's really meant to be used once in the application, no more.

Thanks for your feedback!

**Pull request merged**